api/v1/views/cities.py

```diff
@@ -0,0 +1,112 @@
+#!/usr/bin/python3
+""" module for city object view"""
+from api.v1.views import app_views
+from flask import Flask, jsonify, abort, request
+from models import storage
+from models.city import City
+from models.state import State
+
+
+@app_views.route("/cities/<city_id>", methods=['GET'],
+                 strict_slashes=False)
+def get_city(city_id):
+    """
+    function to get city instance by id using get verb
+    args:
+        city_id: id of city we want
+    return:
+        city instance, else 404
+    """
+    city = storage.get(City, city_id)
+
+    if city:
+        return jsonify(city.to_dict())
+    else:
+        abort(404)
+
+@app_views.route("/cities/<city_id>", methods=['DELETE'],
+                 strict_slashes=False)
+def del_city(city_id):
+    """
+    delete city instance with id, using delete verb
+    arg:
+        city_id: id of city we want to delete
+    return:
+        ok status (200) and empty dictionary
+    """
+    city = storage.get("City", city_id)
```

Comment:
Don't forget what we learned from the problem with Guillaume's wording of task 3: get is meant to be fed the class, not the class name. So here `"City"` changes to `City`.